website/pages/api-docs/secret/mongodbatlas/index.mdx

```diff
@@ -0,0 +1,249 @@
+---
+layout: api
+page_title: MongoDB Atlas - Secrets Engines - HTTP API
+sidebar_title: MongoDB Atlas
+description: |-
+  The MongoDB Atlas Secrets Engine for Vault generates MongoDB Atlas Programmatic API Keys dynamically.
+---
+
+# MongoDB Atlas Secrets Engine
+
+The MongoDB Atlas Secrets Engine generates Programmatic API keys for MongoDB Atlas. This allows one to manage the lifecycle of these MongoDB Atlas secrets programmatically. The created MongoDB Atlas secrets are
+time-based and are automatically revoked when the Vault lease expires, unless renewed. Vault will create a Programmatic API key for each lease scoped to the MongoDB Atlas project or organization denoted with the included role(s). An IP Whitelist may also be configured for the Programmatic API key with desired IPs and/or CIDR blocks.
+
+The MongoDB Atlas Programmatic API Key Public and
+Private Key is returned to the caller. To learn more about Programmatic API Keys visit the [Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api/apiKeys/).
+
+## Configure Connection
+
+In addition to the parameters defined by the Secrets Engines Backend, this plugin has a number of parameters to further configure a connection.
+
+| Method   | Path                         |
+| :--------------------------- | :--------------------- |
```

Comment:
Nit: it would be nice if all the middle horizontal bars in this doc aligned with the above and below, just makes it easier to read.

Reply:
updated